src/asm2wasm.h

```diff
@@ -206,6 +218,78 @@ struct Asm2WasmPreProcessor {
       *marker = START_FUNCS[0];
     }
 
+    // handle debug info, if this build wants that.
+    if (debugInfo) {
+      // asm.js debug info comments look like
+      //   ..command..; //@line 4 "tests/hello_world.c"
+      // we convert those into emscripten_debuginfo(file, line)
+      // calls, where the params are indices into a mapping. then
+      // the compiler and optimizer can operate on them. after
+      // that, we can apply the debug info to the wasm node right
+      // before it - this is guaranteed to be correct without opts,
+      // and is usually decently accurate with them.
+      auto size = strlen(input);
+      auto upperBound = Index(size * 1.25) + 100;
```

Comment:
Magic numbers are magic :)

Does this 100 match the 100 a few lines down?
Would be good to replace these with named constants for clarity.